huaweicloud/services/iec/data_source_huaweicloud_iec_vpc.go

```diff
@@ -47,43 +50,50 @@ func DataSourceIECVpc() *schema.Resource {
 	}
 }
 
-func dataSourceIECVpcRead(d *schema.ResourceData, meta interface{}) error {
-	config := meta.(*config.Config)
-	iecClient, err := config.IECV1Client(GetRegion(d, config))
+func dataSourceIECVpcRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	iecClient, err := cfg.IECV1Client(cfg.GetRegion(d))
 	if err != nil {
-		return fmtp.Errorf("Error creating HuaweiCloud IEC client: %s", err)
+		return diag.Errorf("error creating IEC client: %s", err)
 	}
 
 	listOpts := vpcs.ListOpts{
 		ID:   d.Get("id").(string),
 		Name: d.Get("name").(string),
 	}
 
-	logp.Printf("[DEBUG] query VPCs using given filter: %+v", listOpts)
+	log.Printf("[DEBUG] query VPCs using given filter: %+v", listOpts)
```

Comment:
```suggestion
	log.Printf("[DEBUG] Query VPCs using given filter: %+v", listOpts)
```
and fix the other resource